Documentation/platforms/container-linux-systemd.md

```diff
@@ -0,0 +1,164 @@
+# Run etcd on Container Linux with systemd
+
+The following guide shows how to run etcd with [systemd][systemd-docs] under [Container Linux][container-linux-docs].
+
+## Provisioning an etcd cluster
+
+Cluster bootstrapping in Container Linux is simplest with [Ignition][container-linux-ignition]; `coreos-metadata.service` dynamically fetches the machine's IP for discovery. Note that etcd's discovery service protocol is only meant for bootstrapping, and cannot be used with runtime reconfiguration or cluster monitoring.
+
+To avoid accidental misconfiguration, [Container Linux Config Transpiler][container-linux-ct] helpfully verifies etcd configurations when generating Ignition files:
```

Comment:
can drop the link/full name if the working example + description goes first:
`To avoid accidental misconfiguration, the transpiler`